### PR TITLE
doc(release): prepare release notes for Centreon MAP 22.10.1

### PR DESCRIPTION
**Pull request by Barbeloa**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [X] 22.10.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -15,7 +15,26 @@ Pour faire des demandes d'évolutions ou reporter des bugs sur les extensions co
...
+Release date:
+
+#### Security fixes
+
+- Actuator endpoints are now disabled by default, except for health and metrics
+- Fix the security issue CVE-2022-42889 (Text4shell)
+
+#### Bug fixes
+
+- Fixed an issue where resources tooltips in MAP legacy widget could not be displayed properly
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:37]:
```suggestion
- Fixed an issue where resources' tooltips in the MAP legacy widget could not be displayed properly
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -16,7 +16,26 @@ If you have feature requests or want to report a bug, please contact support.
...
+Release date:
+
+#### Security fixes
+
+- Actuator endpoints are now disabled by default, except for health and metrics
+- Fix the security issue CVE-2022-42889 (Text4shell)
+
+#### Bug fixes
+
+- Fixed an issue where resources tooltips in MAP legacy widget could not be displayed properly
```
Comment (cg-tw) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:38]:
```suggestion
- Fixed an issue where resources' tooltips in the MAP legacy widget could not be displayed properly
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -15,7 +15,26 @@ Pour faire des demandes d'évolutions ou reporter des bugs sur les extensions co
 
 Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https://www.centreon.com/blog/centreon-22-10-les-nouveautes).
 
-## Centreon MAP
+## Centreon MAP 
+
+### 22.10.1
+
+- First release
+
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:23]: Web editor
New Server
Migration process

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -16,7 +16,26 @@ If you have feature requests or want to report a bug, please contact support.
 
 Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blog/centreon-fall22-whats-new-in-the-22-10-software-version/).
 
-## Centreon MAP
+## Centreon MAP 
+
+### 22.10.1
+
+- First release
+
```
Comment (nmagniezCentreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:24]: Web editor
New Server
Migration process

**Review by nmagniezCentreon — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 5]
```diff
@@ -17,6 +17,29 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
 
 ## Centreon MAP
 
+### 22.10.1
+
+- First release: `<release_date>`
+
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:23]:
```suggestion

> The new MAP extension is now available in a full web version with a new server, under the name of MAP. The former version, including desktop client and associated server, becomes MAP Legacy.

```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 6]
```diff
@@ -18,6 +18,29 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
 
 ## Centreon MAP
 
+### 22.10.1
+
+- First release: `<release_date>`
+
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:24]:
```suggestion

> The new MAP extension is now available in a full web version with a new server, under the name of MAP. The former version, including desktop client and associated server, becomes MAP Legacy.

```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 7]
```diff
@@ -17,6 +17,29 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+Web editor: Create and edit your views directly from your web browser
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:24]:
```suggestion
- Web editor: Create and edit your views directly from your web browser.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 8]
```diff
@@ -17,6 +17,29 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+New server: Brand new server and data model providing better performance
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:25]:
```suggestion
- New server: Brand new server and data model providing better performance.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 9]
```diff
@@ -17,6 +17,29 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+Migration process: Integrated migration process of your legacy views
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:26]:
```suggestion
- Migration process: Integrated migration process of your legacy views.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 10]
```diff
@@ -18,6 +18,29 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+Web editor: Create and edit your views directly from your web browser
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:25]:
```suggestion
- Web editor: Create and edit your views directly from your web browser.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 11]
```diff
@@ -18,6 +18,29 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+New server: Brand new server and data model providing better performance
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:26]:
```suggestion
- New server: Brand new server and data model providing better performance.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 12]
```diff
@@ -18,6 +18,29 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+Migration process: Integrated migration process of your legacy views
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:27]:
```suggestion
- Migration process: Integrated migration process of your legacy views.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 13]
```diff
@@ -17,6 +17,31 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
 
 ## Centreon MAP
 
+### 22.10.1
+
+- First release: `<release_date>`
+
+> The new MAP extension is now available in a full web version with a new server, under the name of MAP. The former version, including desktop client and associated server, becomes MAP Legacy.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:24]:
```suggestion
The new MAP extension is now available in a full web version with a new server, under the name of MAP. The former version, including desktop client and associated server, becomes MAP Legacy.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 14]
```diff
@@ -18,6 +18,31 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
 
 ## Centreon MAP
 
+### 22.10.1
+
+- First release: `<release_date>`
+
+> The new MAP extension is now available in a full web version with a new server, under the name of MAP. The former version, including desktop client and associated server, becomes MAP Legacy.
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:25]:
```suggestion
The new MAP extension is now available in a full web version with a new server, under the name of MAP. The former version, including desktop client and associated server, becomes MAP Legacy.
```

**Review by nmagniezCentreon — DISMISSED** [review]

**Review by cg-tw — DISMISSED** [review]

**Review by cg-tw — DISMISSED** [review]

**Review by smau-centreon — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 15]
```diff
@@ -17,6 +17,31 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+## Centreon MAP Legacy
+
+### 22.10.1
+
+Release date: `November 29, 2022`
+
+#### Security fixes
+
+- Actuator endpoints are now disabled by default, except for health and metrics
+- Fix the security issue CVE-2022-42889 (Text4shell)
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:39]:
```suggestion
- Fixed the security issue CVE-2022-42889 (Text4shell).
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 16]
```diff
@@ -17,6 +17,31 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+
+## Centreon MAP Legacy
+
+### 22.10.1
+
+Release date: `November 29, 2022`
+
+#### Security fixes
+
+- Actuator endpoints are now disabled by default, except for health and metrics
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:38]:
```suggestion
- Actuator endpoints are now disabled by default, except for health and metrics.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 17]
```diff
@@ -17,6 +17,31 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
...
+- Fix the security issue CVE-2022-42889 (Text4shell)
+
+#### Bug fixes
+
+- Fixed an issue where resources' tooltips in the MAP legacy widget could not be displayed properly
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md:43]:
```suggestion
- Fixed an issue where resources' tooltips in the MAP legacy widget could not be displayed properly.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 18]
```diff
@@ -18,6 +18,31 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+
+## Centreon MAP Legacy
+
+### 22.10.1
+
+Release date: `November 29, 2022`
+
+#### Security fixes
+
+- Actuator endpoints are now disabled by default, except for health and metrics
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:39]:
```suggestion
- Actuator endpoints are now disabled by default, except for health and metrics.
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 19]
```diff
@@ -18,6 +18,31 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+- Fix the security issue CVE-2022-42889 (Text4shell)
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:40]:
```suggestion
- Fixed the security issue CVE-2022-42889 (Text4shell).
```

File: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md [hunk 20]
```diff
@@ -18,6 +18,31 @@ Read more about version 22.10 in our [blog post](https://www.centreon.com/en/blo
...
+
+#### Bug fixes
+
+- Fixed an issue where resources' tooltips in the MAP legacy widget could not be displayed properly
```
Comment (smau-centreon) [inline versioned_docs/version-22.10/releases/centreon-commercial-extensions.md:44]:
```suggestion
- Fixed an issue where resources' tooltips in the MAP legacy widget could not be displayed properly.
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by nmagniezCentreon — APPROVED** [review]

**Pull request merged**